api/tests/integration/domain/usecases/get-participants-division_test.js

```diff
@@ -0,0 +1,34 @@
+const { expect, databaseBuilder, catchErr } = require('../../../test-helper');
+const { getParticipantsDivision } = require('../../../../lib/domain/usecases/index');
+const { ForbiddenAccess } = require('../../../../lib/domain/errors');
+
+describe('Integration | UseCase | get-participants-division', () => {
+  context('when the use has access to the campaign', () => {
+    it('returns the participats division', async () => {
```

Comment:
typo : `returns the participants division`

Reply:
Done